spec/controllers/v2/policies_controller_spec.rb

```diff
@@ -173,6 +173,21 @@
         expect(response).to have_http_status :not_acceptable
       end
     end
+
+    context 'relations' do
+      let(:relation_count) { 2 }
+      let(:item) do
+        FactoryBot.create(:v2_policy, :with_hosts, host_count: relation_count, account: current_user.account)
+      end
+      let(:item_relations) { item.reload.hosts }
+      let(:new_relation_count) { 3 }
+      let(:new_relation_list) { FactoryBot.create_list(:host, new_relation_count).map(&:id) }
+      let(:data) { { hosts: new_relation_list } }
+      let(:unrelated_item) { FactoryBot.create(:v2_policy, account: current_user.account) }
+      let(:unrelated_item_relations) { unrelated_item.reload.hosts }
+
```

Comment:
Do we really need to create all these here? Could you move at least the redundant ones under the shared example?

Reply:
Also the variable naming is not the most intuitive, we can work on it after they are moved.